src/actions/types.js

```diff
@@ -9,7 +9,7 @@ import type {
   SourceText,
   Frame,
   Why,
-} from "../types";
+} from "debugger-html";
```

Comment:
I like moving the reference to the module. We should do this in other places we reference `src/types`. There's no code duplication, it's just indirect